aws/resource_aws_eip.go

```diff
@@ -98,6 +105,10 @@ func resourceAwsEipCreate(d *schema.ResourceData, meta interface{}) error {
 		Domain: aws.String(domainOpt),
 	}
 
+	if v := d.Get("public_ipv4_pool"); v != nil && v.(string) != "" {
```

Comment:
This can be shortened to the below. 👍 

```suggestion
	if v, ok := d.GetOk("public_ipv4_pool"); ok {
```

Reply:
Thanks, done.